packages/nuxi/src/commands/test.ts

```diff
@@ -0,0 +1,27 @@
+import consola from 'consola'
+import { loadKit } from '../utils/kit'
+import { defineNuxtCommand } from './index'
+
+export default defineNuxtCommand({
+  meta: {
+    name: 'test',
+    usage: 'npx nuxi test',
+    description: 'Run tests'
+  },
+  async invoke (argv) {
+    process.env.NODE_ENV = process.env.NODE_ENV || 'production'
+    const rootDir = process.cwd()
+
+    consola.warn('`nuxt test` is expiremental')
```

Comment:
(what's the point of mentioning this btw? Command is basically starting vitest) 

```suggestion
    consola.warn('`nuxi test` command is expiremental')
```

Reply:
Also it is "experimental"